lib/ukcpio/cpio.c

```diff
@@ -81,7 +128,7 @@ extract_file(const char *path, const char *contents, size_t len,
 	while (len) {
 		ssize_t written = uk_syscall_r_write(fd, contents, len);
```

Comment:
See, yet another one :yum: . Could it be that we could replace this loop with a reusable `write_all`-like function?

Reply:
Yet another what?
Either way, such a function falls outside the scope of this PR.

Reply:
You are right, I understand I may seem like I am overstepping the PR's boundaries. I may have made myself misunderstood. What I meant is: "Can you move this `while` in a separate `file_write` function in `cpio.c` that maybe other functions could reuse as well?"